src/mds/MDSDaemon.cc

```diff
@@ -340,7 +340,8 @@ void MDSDaemon::set_up_admin_socket()
 				     asok_hook,
 				     "dump metadata cache for subtree");
   ceph_assert(r == 0);
-  r = admin_socket->register_command("dump loads",
+  r = admin_socket->register_command("dump loads "
+                                     "name=depth,type=CephInt,range=0,req=false",
```

Comment:
I think the `depth` parameter should be mandatory.
I know its simple to have `req=false` and be helpful to the user, but the mandatory requirement forces the user to think about the consequences of the action explicitly and act as a reminder.